7_drivers_munge.yml

```diff
@@ -0,0 +1,39 @@
+target_default: 7_drivers_munge
+
+include:
+  - 6_drivers_fetch.yml
+
+packages:
+  - scipiper
+  - dplyr
+  - stringr
+
+sources:
+  - 6_drivers/src/nldas_feather_utils.R
+  - 7_drivers_munge/src/GLM_driver_utils.R
+
+targets:
+  7_drivers_munge:
+    depends:
+      - "7_drivers_munge/out/7_drivers_munge_tasks.ind"
+
+  meteo_file_dir:
+    command: c(I("7_drivers_munge/out"))
+
+  cell_group_files:
+    command: calc_cell_group_files(nldas_cells, replete_time_range, ind_dir = cell_ind_dir) # where is the side-effect of writing to '6_drivers/in/nldas_times.rds'?? (updates cubes)
```

Comment:
Is this comment/question still valid, and if so, is it something you want to discuss, and if so, could you elaborate?

Reply:
OK, happy to discuss this in FY19.